test/ratings/create.rating.test.js

```diff
@@ -0,0 +1,225 @@
+const supertest = require('supertest');
+const { factory } = require('factory-girl');
+const app = require('../../app');
+const { find } = require('../helpers/body');
+const { weetErrors } = require('../factory/weet_factory');
+const { defaultPassword, userErrors } = require('../factory/user_factory');
+
+const request = supertest(app);
+describe('/weets/:id/rating POST', () => {
+  test('Create rating should be success and must increase the user score by 1', async () => {
+    const weet = await factory.create('WeetNew');
+    const user = await factory.create('UserWithHash');
+    const score = 1;
+    const loginRequest = {
+      email: user.email,
+      password: defaultPassword
+    };
+    const loginResponse = await request.post('/users/sessions').send(loginRequest);
```

Comment:
In my opinion, this is a bad practice. What happen if the users login endpoint fails? All your test will fail too. And that is not something true because the rating test will fail but not by his fault.